packages/reporter/src/runnables/runnables.tsx

```diff
@@ -49,15 +49,19 @@ const RunnablesEmptyState = ({ spec, eventManager = events }: RunnablesEmptyStat
       <p>Cypress could not detect tests in this file.</p>
       { !isAllSpecs && (
         <>
-          <FileOpener fileDetails={{
+          <OpenFileInIDE fileDetails={{
             column: 0,
             line: 0,
             originalFile: spec.relative,
             relativeFile: spec.relative,
             absoluteFile: spec.absolute,
           }}>
-            <h3><OpenIcon />Open file in IDE</h3>
-          </FileOpener>
+            <a href="#" onClick={(event) => {
+              event.preventDefault()
+            }}>
+              <h3><OpenIcon />Open file in IDE</h3>
+            </a>
+          </OpenFileInIDE>
```

Comment:
I'm assuming that we don't want a tooltip here either, but we do want the link styles. So this guy is a little cobbled together.

Reply:
I think this is okay for now :+1: